home/management/commands/applications_open_email.py

```diff
@@ -0,0 +1,71 @@
+from django.conf import settings
+from django.core.mail import send_mail
+from django.core.management.base import BaseCommand
+from django.template.loader import render_to_string
+from django.utils import timezone
+
+from accounts.models import CustomUser
+from home.models import Session
+
+
+class Command(BaseCommand):
+    help = """
+    Checks if applications are open for a program session 
+    and notify interested folk via email.
+    To be ran once a day.
+    """
+
+    def handle(self, *args, **options):
+        try:
+            session_applications_starting_today = Session.objects.get(
+                application_start_date=timezone.now().date()
+            )
+        except Session.DoesNotExist:
+            self.stdout.write(
+                self.style.SUCCESS(
+                    "There are no sessions with applications starting today"
+                )
+            )
+            return False
+        emails_sent = 0
+        for user in (
+            CustomUser.objects.select_related("profile")
```

Comment:
Do we need to select profile here? I don't see it being used in the formation of `email_data`.

Reply:
`user.profile.create_unsubscribe_link()`

Reply:
D'oh. Thank you!